.svgo.yml

```diff
@@ -33,6 +33,11 @@ plugins:
   - removeUnknownsAndDefaults:
       keepRoleAttr: true
 
+  # remove any unnecessary attributes
```

Comment:
For clarity, do you mind reordering this so that the `removeUnknownsAndDefaults` configuration comes after this comment as well?

Reply:
Good idea! I've done so. Thanks for the valuable comments :)